File: src/main/java/com/wishai/xzrtw/config/InterceptorRegister.java
package com.wishai.xzrtw.config;

import com.wishai.xzrtw.inteceptor.UserHandlerInterceptor;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;

@Configuration
public class InterceptorRegister extends WebMvcConfigurerAdapter {
    @Override
    public void addInterceptors(InterceptorRegistry registry) {
        registry.addInterceptor(new UserHandlerInterceptor())
                .addPathPatterns("/manager/**");
    }
}

File: src/main/java/com/wishai/xzrtw/config/InterceptorRegister.kt
package com.wishai.xzrtw.config

import com.wishai.xzrtw.inteceptor.UserHandlerInterceptor
import org.springframework.context.annotation.Configuration
import org.springframework.web.servlet.config.annotation.InterceptorRegistry
import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter

@Configuration
class InterceptorRegister : WebMvcConfigurerAdapter() {
    override fun addInterceptors(registry: InterceptorRegistry?) {
        registry!!.addInterceptor(UserHandlerInterceptor())
                .addPathPatterns("/manager/**")
    }
}
